### ISSUE TITLE
BertEmbedding output nan inf

### ISSUE THREAD

**Issue by JrJessyLuo** [thunlp/ERNIE#38]
I tried to get the Ernie embedding for my dataset. But When I use the BertModel, I found that after a batch, the word embedding in BertEmbedding becomes full of nan and inf, which was initialized normally.
I tried to minimize the batch size and learning rate, but it failed.  I don't know how to solve it.

**Comment by zzy14** [comment]
Maybe there are some instances making the loss nan and the nan is spreading during the backpropagation. You can give more detail about your experiment.
